trollflow2/launcher.py

```diff
@@ -441,3 +443,61 @@ def sendmail(config, trace):
     pid = Popen([sendmail, "-t", "-oi"], stdin=PIPE)
     pid.communicate(msg.as_bytes())
     pid.terminate()
+
+
+def launch(args_in):
+    """Launch the processing."""
+    args = parse_args(args_in)
+
+    log_config = args.pop("log_config", None)
+    if log_config is not None:
+        with open(log_config) as fd:
+            log_config = yaml.safe_load(fd.read())
```

Comment:
Can we put this in its own function for clarity?

Reply:
Done in 95d56a1